<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewFieldDialog</class>
 <widget class="QDialog" name="NewFieldDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>新建字段</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>表 名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>字 段：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>别 名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>类 型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>描 述：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tableName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_field">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_alias">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_description">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>28</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pbt_Confirm">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbt_Cancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBox_Type">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <item>
    <property name="text">
     <string>Bool</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SByte</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Byte</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Int16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UInt16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Int32</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UInt32</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Int64</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UInt64</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Float</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Double</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DateTime</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>String</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
